app/kumactl/cmd/uninstall/uninstall_transparent_proxy.go

```diff
@@ -66,7 +62,9 @@ func newUninstallTransparentProxy() *cobra.Command {
 				fmt.Fprintln(cfg.RuntimeStdout, string(content))
 			}
 
-			fmt.Fprintln(cfg.RuntimeStdout, "Transparent proxy cleaned up successfully")
+			if !initializedConfig.DryRun {
+				initializedConfig.Logger.InfoWithoutPrefix("transparent proxy cleanup completed successfully")
```

Comment:
How do I pick `Info` vs `InfoWithoutPrefix` when I want to log something?

Reply:
Actually here it should be just `Info`, but basically `InfoWithoutPrefix` is used when you want to log for example iptables rules, so it's easy to copy and paste them without need of do any modifications